main.go

```diff
@@ -86,6 +107,15 @@ func main() {
 	flag.BoolVar(&enableLeaderElection, "enable-leader-election", false,
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
+	flag.StringVar(&namespace, "namespace", "", "Namespace that the StorageOS components, including api-manager, are installed into.  Will be auto-detected if unset.")
+	flag.StringVar(&schedulerName, "scheduler-name", "storageos", "Name of the Pod scheduler to use for Pods with StorageOS volumes.  Set to an empty value to disable setting the Pod scheduler.")
+	flag.StringVar(&webhookServiceName, "webhook-service-name", "storageos-scheduler-webhook", "Name of the webhook service.")
+	flag.StringVar(&webhookServiceNamespace, "webhook-service-namespace", "", "Namespace of the webhook service.  Will be auto-detected or value of -namespace unset.")
+	flag.StringVar(&webhookSecretName, "webhook-secret-name", "storageos-webhook-secret", "Name of the webhook secret.")
+	flag.StringVar(&webhookSecretNamespace, "webhook-secret-namespace", "", "Namespace of the webhook secret.  Will be auto-detected or value of -namespace unset.")
+	flag.StringVar(&webhookConfigMutatingName, "webhook-config-mutating", "storageos-mutating-webhook", "Name of the mutating webhook configuration.")
+	flag.StringVar(&webhookMutatePodsPath, "webhook-mutate-pods-path", "/mutate-pods", "URL path of the Pod mutating webhook.")
+	flag.DurationVar(&webhookCertRefreshInterval, "webhook-cert-refresh-interval", 3*30*24*time.Hour, "Frequency of webhook certificate refresh.")
```

Comment:
We decided to do this often, maybe about every 30 minutes or less, to reduce the amount of time multiple manager could be out of sync, if that ever happens.

Reply:
Set to 30 minutes